x/bank/types/params.go

```diff
@@ -3,31 +3,139 @@ package types
 import (
 	"fmt"
 
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	yaml "gopkg.in/yaml.v2"
 )
 
 const (
 	// DefaultParamspace for params keeper
 	DefaultParamspace = ModuleName
-	// DefaultSendEnabled enabled
-	DefaultSendEnabled = true
 )
 
-// ParamStoreKeySendEnabled is store's key for SendEnabled
-var ParamStoreKeySendEnabled = []byte("sendenabled")
+var (
+	// KeySendEnabled is store's key for SendEnabled Params
+	KeySendEnabled = []byte("SendEnabled")
+	// KeyDefaultSendEnabled is store's key for the DefaultSendEnabled option
+	KeyDefaultSendEnabled = []byte("DefaultSendEnabled")
+)
 
-// ParamKeyTable type declaration for parameters
+// ParamKeyTable for bank module.
 func ParamKeyTable() paramtypes.KeyTable {
-	return paramtypes.NewKeyTable(
-		paramtypes.NewParamSetPair(ParamStoreKeySendEnabled, false, validateSendEnabled),
-	)
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+// NewParams creates a new parameter configuration for the bank module
+func NewParams(defaultSendEnabled bool, sendEnabledParams SendEnabledParams) Params {
+	return Params{
+		SendEnabled:        sendEnabledParams,
+		DefaultSendEnabled: defaultSendEnabled,
+	}
+}
+
+// DefaultParams is the default parameter configuration for the bank module
+func DefaultParams() Params {
+	return Params{
+		SendEnabled: SendEnabledParams{},
+		// The default send enabled value allows send transfers for all coin denoms
+		DefaultSendEnabled: true,
+	}
+}
+
+// Validate all bank module parameters
+func (p Params) Validate() error {
+	if err := validateSendEnabledParams(p.SendEnabled); err != nil {
+		return err
+	}
+	return nil
```

Comment:
```suggestion
	return validateIsBool(p. DefaultSendEnabled)
```

Reply:
Added missing validation for `DefaultSendEnabled` property of `Params` object per suggestion.